### DISCUSSION TITLE
Disable certain nodes?

### DISCUSSION THREAD

**Discussion by half0wl** [aguingand/tiptap-markdown#53] [Q&A] [answered]
I'd like to expose very basic formatting (bold, italic, etc.) and nothing else. Nodes such as code blocks, headings, etc. should be rendered as-is in plain-text (`# foo`, ...)

Is this possible?

**Comment by aguingand** [comment, marked as answer]
You can't render markdown as-is text in tiptap because it must be html tags. So disabling markdown for element would result to `<h1>foo</h1>`
